crates/miden-testing/src/mock_chain/chain_builder.rs

```diff
@@ -395,26 +396,45 @@ impl MockChainBuilder {
         Ok(())
     }
 
-    // NOTE METHODS
+    // NOTE ADD METHODS
     // ----------------------------------------------------------------------------------------
 
     /// Adds the provided note to the initial chain state.
     pub fn add_note(&mut self, note: impl Into<OutputNote>) {
         self.notes.push(note.into());
     }
 
-    /// Creates a new P2ANY note from the provided parameters and adds it to the list of genesis
-    /// notes. This note is similar to a P2ID note but can be consumed by any account.
+    /// Creates a new public P2ANY note from the provided parameters and adds it to the list of
+    /// genesis notes.
+    ///
+    /// This note is similar to a P2ID note but can be consumed by any account.
     ///
     /// In the created [`MockChain`], the note will be immediately spendable by `target_account_id`
     /// and carries no additional reclaim or timelock conditions.
-    pub fn add_p2any_note(
+    pub fn add_public_p2any_note(
         &mut self,
         sender_account_id: AccountId,
         asset: impl IntoIterator<Item = Asset>,
     ) -> anyhow::Result<Note> {
-        let note = create_p2any_note(sender_account_id, asset);
+        let note = self.create_p2any_note(sender_account_id, NoteType::Public, asset)?;
+        self.add_note(OutputNote::Full(note.clone()));
+
+        Ok(note)
+    }
 
+    /// Creates a new private P2ANY note from the provided parameters and adds it to the list of
+    /// genesis notes.
+    ///
+    /// This note is similar to a P2ID note but can be consumed by any account.
+    ///
+    /// In the created [`MockChain`], the note will be immediately spendable by `target_account_id`
+    /// and carries no additional reclaim or timelock conditions.
+    pub fn add_private_p2any_note(
+        &mut self,
+        sender_account_id: AccountId,
+        asset: impl IntoIterator<Item = Asset>,
```

Comment:
nit: This was like this from before, but should these (this applies to various related functions here) parameters be named `assets` instead?